contracts/margin/external/ERC20/ERC20PositionWithdrawer.sol

```diff
@@ -90,13 +90,13 @@ contract ERC20PositionWithdrawer is ReentrancyGuard
         returns (uint256, uint256)
     {
         // withdraw tokens
+        address withdrawnToken = ERC20Position(erc20Position).heldToken();
```

Comment:
Was this change just for clarity?

Reply:
Changing back, not necessary to do that call/read if 0 tokens are returned